<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userView</class>
 <widget class="QWidget" name="userView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgba(248, 253, 255, 206);
color:rgb(1, 31, 54);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="userView_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>117</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#profile_widget{border:1px solid rgb(48, 162, 249)}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="img_label">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>95</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>95</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources/resources.qrc">:/images/User-Profile-PNG-Image.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="firstname_label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Firstname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lastname_label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Lastname</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="headline_plainTextEdit">
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>Headline...
</string>
          </property>
          <property name="maximumBlockCount">
           <number>0</number>
          </property>
          <property name="backgroundVisible">
           <bool>false</bool>
          </property>
          <property name="centerOnScroll">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
  <include location="../../../../../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
